api/src/services/verification.ts

```diff
@@ -64,8 +64,18 @@ ON CONFLICT DO NOTHING;`;
 
 const CONTRACT_VERIFICATION_STATUS = 'SELECT * FROM verification_request WHERE address = $1;';
 
+const BLOCK_FINALIZED_MS = 40000;
```

Comment:
This hack wont work because block finality time is not constant

Reply:
we'll need a gql subscription then